framework/package.json

```diff
@@ -137,18 +137,11 @@
 		"rx": "4.1.0",
 		"sinon": "7.2.2",
 		"sinon-chai": "3.3.0",
-		"snyk": "^1.122.0",
 		"stampit": "4.2.0",
 		"supertest": "3.3.0"
 	},
 	"lisk": {
 		"minVersion": "1.1.0-rc.0",
 		"protocolVersion": "1.0"
-	},
-	"husky": {
-		"hooks": {
-			"pre-commit": "lint-staged",
```

Comment:
Why remove these hooks?

Reply:
This line was running the linting commands from https://github.com/okonet/lint-staged. The dependency check was in `pre-push`

```
			"pre-push": "eslint $(git ls-files '*.js') && npm run check:dependencies"
```

Reply:
yes, but hooks are managed in the root level now, and not required here anymore